server/models.py

```diff
@@ -37,9 +37,22 @@ def __repr__(self):
         return f"<id {self.id}>"
 
     def json(self) -> Dict[str, str]:
+
+        if debug:

```

Comment:
This if-statement will always happen as the debug variable is set to true, so the github field will always be hardcoded, can you update this?